### DISCUSSION TITLE
Make the save button only "save"

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#5259] [Feature Requests]
Is it maybe an idea to make the save button only reload the record?
Like when you have a hook that wil fill in data for you, you would have to save and then search your record again.

**Comment by benhaynes** [comment]
You can hit "META-S" (eg: Command-S), which is also available in the "..." Save Context Menu. We could look into this having an option for default behavior in the future.

**Reply by Attacler** [reply]
alright thank you so far!

**Comment by rijkvanzanten** [comment]
This is what the save-and-stay option does under the save submenu and what `Ctrl+S` / `CMD+S` is mapped to 👍🏻

**Reply by benhaynes** [reply]
🧠⚡🧠

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
